mne/time_frequency/tfr.py

```diff
@@ -742,9 +745,26 @@ def plot_topo(self, picks=None, baseline=None, mode='mean', tmin=None,
         border : str
             matplotlib borders style to be used for each sensor plot.
         fig_facecolor : str | obj
-            The figure face color. Defaults to black.
-        font_color: str | obj
-            The color of tick labels in the colorbar. Defaults to white.
+            The figure face color. Defaults to black. If "black", spines are
+            white (else they are black).
+        font_color : str | obj
+            The color of text in the colorbar and title. Defaults to
+            spine_color (which is automatically set based on fig_facecolor
+            if None).
+        external_legend : bool | float
```

Comment:
Well, you originally complained the legend should have the same size as the plots. This is now the default, but IMO gives suboptimal results in some cases, so I've added the option, for these cases, to upscale the legend. So the user doesn't have to scale it, but if they desire a legend larger than plot size, there is an option for that. But I guess having the simplicity of only one keyword for legends might be superior.